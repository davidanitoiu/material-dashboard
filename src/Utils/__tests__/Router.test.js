import React from 'react';
import AppRouter from '@utils/Router';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

describe('React Router ->', () => {
    it('When initializing the Router, Sidebar component is shown #reactRouter', () => {
        const wrapper = shallow(
            <AppRouter />
        );
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    })
})
