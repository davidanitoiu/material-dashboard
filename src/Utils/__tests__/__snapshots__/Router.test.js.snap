// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React Router -> When initializing the Router, Sidebar component is shown #reactRouter 1`] = `
<BrowserRouter>
  <Connect(Sidebar) />
  <Switch>
    <Route
      exact={true}
      path="/"
    >
      <Redirect
        push={true}
        to="/dashboard"
      />
    </Route>
    <Route
      component={[Function]}
      path="/icons"
    />
    <Route
      component={[Function]}
      path="/maps"
    />
    <Route
      component={[Function]}
      path="/notifications"
    />
    <Route
      component={[Function]}
      path="/profile"
    />
    <Route
      component={[Function]}
      path="/tableList"
    />
    <Route
      component={[Function]}
      path="/typography"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(Dashboard)",
          "type": [Function],
        }
      }
      path="/dashboard"
    />
  </Switch>
</BrowserRouter>
`;
